src/enums.ts

```diff
@@ -30,9 +30,12 @@ export function generateEnum(
 
   enumDesc.value.forEach((valueDesc, index) => {
     const info = sourceInfo.lookup(Fields.enum.value, index);
-    maybeAddComment(info, chunks, valueDesc.options?.deprecated, `${valueDesc.name} - `);
+    const valueName = options.removeEnumPrefix
+      ? valueDesc.name.replace(`${camelToSnake(fullName)}_`, '')
+      : valueDesc.name;
```

Comment:
I think we do this logic in a few places, maybe add a helper method like `getValueName(ctx, valueDesc)` that internally does the `options.removeEnumPrefix` check?